web.py

```diff
@@ -4,17 +4,34 @@
 from github import Github
 from joiner_scripts.joiner import AqGpsJoiner
 import datetime
+import base64
 
 app = Flask(__name__)
 
 APP_ROOT = os.path.dirname(os.path.abspath(__file__))
 
+#Variables
+g = Github("fa0dfb5e6103c9bf91cbbcc22e3c1aa5708cbdb4")
+repo = g.get_user().get_repo('woaq')
+
 @app.route('/')
```

Comment:
Views should be kept in separate views.py file. To the maximum extend possible, they should only execute routing. Any related functions should be called from other files.